plugin/pkg/admission/broker/authsarcheck/admission.go

```diff
@@ -29,6 +29,7 @@ import (
 	"k8s.io/apiserver/pkg/admission"
 	kubeclientset "k8s.io/client-go/kubernetes"
 
+	scv "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/validation"
```

Comment:
just curious why not just use validation here. Ie, why scv? I find it less readable down below.

Reply:
I don't think that we should be pulling in validation here.  Just ignore the case where the secret isn't defined and let validation error it out later.

Reply:
The import line was just copied, but the custom package name can be dropped.

The reason the validation is pulled in here is to stop a crash when the spec isn't populated like this:
```
apiVersion: servicecatalog.k8s.io/v1beta1
kind: ClusterServiceBroker
metadata:
  name: abroker
spec:
  url: http://abroker.svc.cluster.local
  authInfo:
    bearer:
      secretRef:
```
I can add more checking for nil or can leverage the existing validation code to do that. If using the validation code is strongly unwanted, I'll port the code over.